### PR TITLE
feat(TMST-618): Breakout section component

### PR DESCRIPTION
**Pull request by lukakereceni**
[TMST-618](https://nidigitalsolutions.jira.com/browse/TMST-618)

**Description**
- Created BreakoutSection component using the NK components
- Wrote unit tests for the BreakoutSection component
- Wrote a story for the BreakoutSection component
- Created mock data

Screenshot 2023-04-18 at 11 23 27

[TMST-618]: https://nidigitalsolutions.jira.com/browse/TMST-618?atlOrigin=eyJpIjoiNTBlM2Y4OTYxZDFiNDY1Zjk3ZjZjMjI3MDQ2ZDZjMDYiLCJwIjoiaiJ9

### CODE REVIEW

File: packages/ts-newskit/package.json [hunk 1]
```diff
@@ -50,9 +50,11 @@
     "node": ">=8.9"
   },
   "dependencies": {
+    "@emotion-icons/material": "3.14.0",
     "@emotion/react": "11.10.0",
     "@emotion/styled": "11.10.0",
-    "newskit": "7.0.0",
+    "@newskit-themes/the-times": "0.0.0",
+    "newskit": "7.1.0",
```
Comment (douglasmik) [inline packages/ts-newskit/package.json:57]: is there a reason why this isn't using the latest version 7.2.0?

File: packages/ts-newskit/src/components/puzzles/breakout-section/breakoutSection.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import { BreakoutSection } from './index';
+import { breakoutItems } from './fixtures/breakoutItems.json'
+
+<Meta
+  title="Newskit Components/Puzzles/BreakoutSection"
+  component={BreakoutSection}
+/>
+
+# BreakoutSection component
+
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/puzzles/breakout-section/breakoutSection.stories.mdx:13]: Is it possible to add a description for this component

File: packages/ts-newskit/src/components/puzzles/breakout-section/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      <BreakoutSectionContainer>
+        <BreakoutSectionTitle as="h2">
+          Add a challenge to your day
+        </BreakoutSectionTitle>
+        <BreakoutSectionSubtitle as="p">
+          Play exclusive Word puzzles, Number puzzles, Quizzes and more
+        </BreakoutSectionSubtitle>
+        <BreakoutSectionGridLayout
+          columns={{ sm: 'repeat(2, 1fr)', lg: 'repeat(3, 1fr)' }}
+          rowGap={{ xs: '20px', sm: '28px', md: '24px' }}
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/puzzles/breakout-section/index.tsx:32]: can we use tokens wherever possible;
https://newskit.co.uk/theme/foundation/spacing/ (may be updated with new theme?)

File: packages/ts-newskit/src/components/puzzles/breakout-section/styles.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  background-color: #ffbb6a;
+  ${getMediaQueryFromTheme('md')} {
+    padding: 40px 28px 40px 24px;
+  }
+`;
+
+export const BreakoutSectionTitle = styled(TextBlock)<TextBlockProps>`
+  font-size: 24px;
+  font-weight: 800;
+  line-height: 36px;
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/puzzles/breakout-section/styles.ts:32]: is there a typography-preset for this, or would it be better to create one?

File: packages/ts-newskit/src/components/puzzles/breakout-section/styles.ts [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 40px 24px 40px 20px;
+  background-color: #ffbb6a;
+  ${getMediaQueryFromTheme('md')} {
+    padding: 40px 28px 40px 24px;
+  }
+`;
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/puzzles/breakout-section/styles.ts:27]: `Stack` component may be better for this as uses flex with props for alignment, etc.

Comment (douglasmik) [reply packages/ts-newskit/src/components/puzzles/breakout-section/styles.ts:27]: also think stack supports props for padding, etc

File: packages/ts-newskit/src/components/puzzles/breakout-section/styles.ts [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  border: 4px solid #01000d;
+  border-radius: 0;
+  background-color: #ffffff;
+  box-shadow: 4px 4px 0 #01000d;
+  z-index: 2;
+  ${getMediaQueryFromTheme('sm')} {
+    flex-direction: row;
+    gap: 40px;
+  }
+`;
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/puzzles/breakout-section/styles.ts:70]: May be better to add button inside `Stack` component?

**Review by jasminaj0 — APPROVED** [review]

File: packages/ts-newskit/src/components/puzzles/personal-stats/index.tsx [hunk 7]
```diff
@@ -45,8 +45,8 @@ export const PersonalStats: FC<PersonalStatsProps> = ({ data }) => {
               <GridLayoutItem justifySelf="center">
                 <TextBlock
                   as="span"
-                  typographyPreset="editorialHeadline040"
-                  stylePreset="inkInverse"
+                  typographyPreset="editorialDisplay004"
```
[the remark this answers is not in the document]
Comment (lukakereceni) [reply packages/ts-newskit/src/components/puzzles/personal-stats/index.tsx:48]: Puzzles theme:
Screenshot 2023-04-27 at 14 28 11

**Review by ibrahim-kurhan — APPROVED** [review]

**Pull request merged**